frontend/src/components/Appbar.tsx

```diff
@@ -27,54 +55,112 @@ const AppbarContainer = styled(Stack)`
   }
 `;
 
-const XIconWrapper = styled(SvgIcon)`
-  svg path {
-    fill: #adb2b8;
-  }
-`;
+function renderBreadcrums(breadcrumsProps: BreadcrumsProps[]) {
+  return (
+    <Stack direction="row" justifyContent="center" alignItems="center">
+      <Breadcrumbs separator={<ArrowLogo fontSize="1rem" />} aria-label="breadcrumb">
+        {breadcrumsProps.map((breadcrumObj, i) => {
+          const Icon = breadcrumObj.icon;
+          if (breadcrumObj.link) {
+            return (
+              <Link
+                key={generateRandString()}
```

Comment:
I know that we discussed about it yesterday, but since we are not re-ordering that list, let's just use the index here (I can elaborate as of why it's preferable to the current implementation in person)

Reply:
ok. done